content/en/real_user_monitoring/connect_rum_and_traces/_index.md

```diff
@@ -47,6 +47,8 @@ Use frontend data from RUM, as well as backend, infrastructure, and log informat
 
 ### Setup RUM
 
+**Note:** Configuring RUM and Traces makes use of RUM paid data in APM, which may impact your billing.
```

Comment:
```suggestion
**Note:** Configuring APM spans to connect to RUM data makes use of RUM paid data in APM, which may impact your billing.
```

Reply:
@jamiemilstein I have a few thoughts/questions:
- This note actually does route to the APM docs page
- Does the update above make it any better/clearer? I'm open to any rephrasing suggestions you have
- I just realized something - would it would be better to move the note to the last paragraph under the **Overview** section?

Reply:
yes i think it's better! i think the only thing I would swap is _APM paid data in RUM_ as we are talking about viewing traces in RUM. 
@hugokash does this make sense from your side of things? since we're talking about billing wanted to get a second set of eyes

Reply:
That's ok. Should we specify "may impact your APM billing" ?

Reply:
@hugokash sure! Thanks for your input as well @jamiemilstein.